providers/src/airflow/providers/google/cloud/sensors/gcs.py

```diff
@@ -42,6 +44,8 @@
     from google.api_core.retry import Retry
 
     from airflow.utils.context import Context
+AIRFLOW_VERSION = Version(airflow_version)
+AIRFLOW_V_3_0_PLUS = Version(AIRFLOW_VERSION.base_version) >= Version("3.0.0")
```

Comment:
This is the correct way to detect version.

Reply:
Changed it